src/hooks/system_mod.rs

```diff
@@ -73,16 +73,16 @@ fn run_command(
         return Err(format!("User denied execution of system command `{full_command}`.").into());
     }
 
-    let output = Command::new(name).args(args).output();
+    let status = Command::new(name).args(args).status();
 
-    match output {
-        Ok(output) => {
-            if output.status.success() {
+    match status {
+        Ok(status) => {
+            if status.success() {
                 Ok(Dynamic::UNIT)
             } else {
                 Err(format!(
                     "System command `{full_command}` returned non-zero status: {}",
```

Comment:
```suggestion
                    "System command `{full_command}` returned non-zero status: {status}",
```

Reply:
You might want to enable https://rust-lang.github.io/rust-clippy/master/#uninlined_format_args to automate this feedback